app/models/patient.rb

```diff
@@ -1,4 +1,8 @@
 class Patient < ApplicationRecord
   has_many :appointments, dependent: :destroy
   has_many :doctors, through: :appointments
+
+  validates :name, presence: true
+  validates :email, presence: true, uniqueness: true
+  validates :password, presence: true, length: { minimum: 6 }, allow_nil: true
```

Comment:
- Please remove these validations .
"I don't think you need to add validation to the Patient Model at this time because authentication hasn't been set up yet. Authentication should be implemented in a separate PR."